### DISCUSSION TITLE
一键脚本部署找不到密码

### DISCUSSION THREAD

**Discussion by ttt789-ppo** [AlistGo/alist#713] [Q&A]
我用的一键脚本装的，装完打开提示这个，我看文档里面只有docker的密码查看命令，在它安装目录看也没有日志，不知道怎么弄，我在alist文件夹里面的config配置文件配置用户名密码也没有用，还是这样提示
![Screenshot_2022-03-09-18-12-46-418_mark via](https://user-images.githubusercontent.com/72877282/157422209-c4c0709f-2be0-415e-b62b-e8f048be3ab0.jpg)

**Comment by xhofe** [comment]
```shell
cd /opt/alist
./alist -password
```
ps: 安装脚本会输出初始密码。
